packages/dapp/src/components/BackgroundGradient/BackgroundGradient.style.tsx

```diff
@@ -80,8 +80,8 @@ export const BackgroundGradients = styled('span')<any>(({ theme }) => ({
 }));
 
 export const BackgroundGradientBottomLeft = styled(BackgroundGradient, {
-  shouldForwardProp: (prop) => prop !== 'isDarkMode',
-})<BackgroundGradientProps>(({ theme, isDarkMode }) => ({
+  shouldForwardProp: (prop) => prop !== 'showWelcome',
```

Comment:
As I see, we don't use it. Should this be removed?

Reply:
Also, if we have `isDarkMode` props elsewhere, we should remove them in favor of `theme.palette.mode === 'dark'`.

Reply:
Great deal, good find and cleaned up!